automatic/windows10-media-creation-tool/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,23 @@
+﻿$ErrorActionPreference = 'Stop'
+$exeName = "MediaCreationTool.exe"
+
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+  PackageName  = $env:chocolateyinstall
+  FileFullPath = "$toolsDir\$exeName"
+  Url          = ''
+  Checksum     = ''
+  ChecksumType = ''
+}
+Get-ChocolateyWebFile @packageArgs
+
+$AppPathKey = "Registry::HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\$($exeName)"
+If (!(Test-Path $AppPathKey)) {New-Item "$AppPathKey" | Out-Null}
+Set-ItemProperty -Path $AppPathKey -Name "(Default)" -Value "$($packageArgs.FileFullPath)" -Force
+Set-ItemProperty -Path $AppPathKey -Name "Path" -Value "$toolsDir" -Force
+
+Write-Output "******************************************************************************************************************"
```

Comment:
I would remove this entire section to be consistent with other packages. This is basic OS usage, its equally valuable to inform users that [x] should be clicked to close the window.